local/check-signature.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+if [[ $# -ne 1 ]]; then
+    echo "usage: $0 <channel>"
+    exit 1
+fi
+channel="$1"
+
+# Work in a temporary directory
+dir="$(mktemp -d)"
+trap "rm -rf ${dir}" EXIT
+cd "${dir}"
+
+mc cp "local/static/dist/channel-rust-${channel}.toml" . >/dev/null
+mc cp "local/static/dist/channel-rust-${channel}.toml.asc" . >/dev/null
```

Comment:
Er, why do we invoke... midnight commander(?) here?

Reply:
`mc` is also Minio's CLI, and I'm using it to download stuff from the minio instance.

Reply:
:+1: okay, maybe worth a comment.